Dev-Corner/slides/toolshed.html

```diff
@@ -0,0 +1,861 @@
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Install a tool from the Tool Shed
+                            - Go to the admin interface and click on "Search Tool Shed"
+                            <img src="../images/admin_searchtoolshed_ss.png" class="no-border" >
+                            - Select a Tool Shed
+
+                            <img src="../images/goToToolshed_ss.png" class="no-border" >
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Install a tool from the Tool Shed
+                            - Search your tool
+                            <img src="../images/searchTool_ss.png" class="no-border" >
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Install a tool from the Tool Shed
+                            - <span style="font-size:70%">Types of repository</span>
+                              - <span style="font-size:55%">Tool (orange): contains tools (tool wrappers)</span>
+                              - <span style="font-size:55%">Tool dependencies (red): contains only the file tool\_dependencies.xml.
+                              !! tool\_dependencies.xml file is no longer supported because of the use of conda!
+                              Required name: package_* </span>
+                              - <span style="font-size:55%">Tool suite (blue): contains only the file repository\_dependencies.xml.
+                              Required name: suite_* </span>
+                            <img src="../images/typeOfRep_ss.png" class="no-border" width="70%" >
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Install a tool from the Tool Shed
+                            <img src="../images/goToToolInstall_ss.png" class="no-border" >
+                            - Install the tool
+                            <img src="../images/launchInstallTool_ss.png" class="no-border" >
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Install a tool from the Tool Shed
+                            - Install the tool
+                            <img src="../images/installTool_ss.png" class="no-border" width="80%">
+                            - <span style="font-size:70%"> Selecting Tool Shed AND conda will make Galaxy to install both</span>
+                            - <span style="font-size:70%"> Recommended: use conda </span>
```

Comment:
I guess we can mention this in our talk or if people ask, no need to add it here.